src/main/java/com/example/demo/models/Team.java

```diff
@@ -0,0 +1,19 @@
+package com.example.demo.models;
+
+import lombok.*;
+
+@Getter
+@Setter
+@NoArgsConstructor
+public class Team {
+    private String name;
+    private String capitan;
+    private String coach;
+    private int wins;
+
+    public Team(String name, String capitan, String coach) {
```

Comment:
You don`t need this. Try to find some other lombok controller (except NoArg and AllArg)